lib/src/opentdf.ts

```diff
@@ -87,6 +92,7 @@ export type CreateNanoTDFOptions = CreateOptions & {
 };
 
 export type CreateNanoTDFCollectionOptions = CreateNanoTDFOptions & {
+  platformUrl: string;
```

Comment:
Can we just use the one from the client? It would seem odd to me that you would use multiple platforms with the same client

Reply:
We end up using the one from the client afaik here:
https://github.com/opentdf/web-sdk/blob/bdb5ba6402503c27016e26f38d6208924e6b6f74/lib/src/opentdf.ts#L358

This is added because otherwise typescript will not allow to pass in the field

Reply:
This is a side effect of using the deprecated nanotdf.Client class, since it has a required type with this name. We can remove that as part of standardizing on the 1.x API, and then remove this as a required parameter from the Collection helper object, or perhaps unify on something more abstract.